lib/local-constructs/dynamodb-table/index.ts

```diff
@@ -0,0 +1,47 @@
+import { Construct } from "constructs";
+import { CfnOutput } from "aws-cdk-lib";
+import * as dynamodb from "aws-cdk-lib/aws-dynamodb";
+
+interface DynamoDBTableProps {
+  readonly stage: string;
+  readonly name: string;
+  readonly partitionKey: { name: string; type: dynamodb.AttributeType };
+  readonly gsi?: {
+    indexName: string;
+    partitionKey: { name: string; type: dynamodb.AttributeType };
+  };
+}
+
+export class DynamoDBTable extends Construct {
+  public readonly table: dynamodb.Table;
+
+  constructor(scope: Construct, id: string, props: DynamoDBTableProps) {
+    super(scope, id);
+
+    this.table = new dynamodb.Table(this, "Table", {
+      tableName: `${props.stage}-${props.name}`,
+      partitionKey: props.partitionKey,
+      billingMode: dynamodb.BillingMode.PAY_PER_REQUEST,
+      stream: dynamodb.StreamViewType.NEW_AND_OLD_IMAGES,
```

Comment:
I don't think all our tables need a stream, but it definitely doesn't hurt? If it's not that way in SEDS it definitely has some in other apps that don't have streams

Reply:
Yeah, the postkafka lambda connects to every dynamodb table's stream.  dataconnect's doesn't use every table's stream, but postkafka has all of them.